include/swift/SIL/SILDeclRef.h

```diff
@@ -162,6 +162,17 @@ struct SILDeclRef {
     AsyncEntryPoint,
   };
 
+  /// Represents the variants of a back deployable function.
+  enum class BackDeploymentKind : unsigned {
+    /// The original function.
```

Comment:
This is the one that is part of the ABI, right?

Reply:
That's right, I'll clarify that.